spaces/S000030/properties/P000055.md

```diff
@@ -5,6 +5,12 @@ value: true
 refs:
 - doi: 10.1007/978-1-4612-6290-9_6
   name: Counterexamples in Topology
+- zb: "1052.54001"
+  name: General Topology (Willard)
 ---
 
+The countable product of {P55} spaces is {P55}
+(see e.g. 24.11 of {{zb:1052.54001}}),
+and {S25|P55}.
+
 See item #7 for space #36 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

Comment:
```suggestion
See item #7 for space #36/#37 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

It appears that #36 and #37 don't have separate listings in *Counterexamples* (because they recognized the homeomorphism) so perhaps it's clearer to use `#36/#37` in our references.